test_scripts/RC/CLIMATE_RADIO/Capabilities/005_Resend_only_supported_parameters.lua

```diff
@@ -50,10 +49,10 @@ local absent_params = {
 
 --[[ Local Functions ]]
 local function setVehicleData(params)
-	local cid = commonRC.getMobileSession():SendRPC("SetInteriorVehicleData", {moduleData = params})
+  local cid = commonRC.getMobileSession():SendRPC("SetInteriorVehicleData", {moduleData = params})
 
   if params.radioControlData.frequencyInteger then
-    EXPECT_HMICALL("RC.SetInteriorVehicleData",	{
+    EXPECT_HMICALL("RC.SetInteriorVehicleData", {
             appID = commonRC.getHMIAppId(1),
```

Comment:
```suggestion
      appID = commonRC.getHMIAppId(1),
```

Not changed by this PR, but for consistency